streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -162,15 +164,16 @@
  * @see OutputVerifier
  */
 @InterfaceStability.Evolving
-public class TopologyTestDriver {
+public class TopologyTestDriver implements Closeable {
```

Comment:
Do we need a KIP for this change? \cc @guozhangwang 

I think the change is good -- we should have done this from the beginning on...

Reply:
I think it would not require a KIP, as it does not affect any public API behaviors that user depends on.